CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 4.8.7 (2021-03-30)
+
+### Bug fixes
+
+* Improve ANR handler compatibility with Google Play reporting mechanism
```

Comment:
Copy/paste error

Reply:
Fixed this by adding previously unstaged changes.